WooCommerce/Classes/ViewRelated/Products/Edit Product/Downloadable Files/File List/BottomSheetListSelector/DownloadableFileFormBottomSheetAction.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+import Yosemite
+
+/// Actions in the downloadable file form bottom sheet to add a new downloadable file.
+enum DownloadableFileFormBottomSheetAction {
+    case fromDevice
+    case fromWordPressMediaLibrary
+    case fromFileURL
+}
```

Comment:
nit: how about `DownloadableFileSource` and so that it's more concise and we can remove the `from` from each case? also, it's not tied to the bottom sheet UX anymore in case we have another way to select a source 😄 

```suggestion
enum DownloadableFileSource {
    case device
    case wordPressMediaLibrary
    case fileURL
}
```